lms/djangoapps/course_api/helpers.py

```diff
@@ -53,9 +53,14 @@ def get_marketing_data(course_key, language):
     marketing_root_format = marketing_link('COURSE_DETAILS_API_FORMAT')
     url = marketing_root_format.format(course_id=course_key)
 
-    response = requests.get(url=url, headers={
-        'Accept-Language': language,
-    })
+    try:
+        response = requests.get(
+            url=url,
+            headers={'Accept-Language': language},
+            timeout=3
```

Comment:
Make it a configurable setting please.

```suggestion
            timeout=settings.MARKETING_API_TIMEOUT,
```

Reply:
Done